src/web/components/Icon/Icon.test.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Icon from './';
+
+describe('Container', () => {
+  let wrapper;
+
+  beforeEach(() => {
+    wrapper = shallow(<Icon>fa-ban</Icon>);
+  });
+
+  it('should render a i with children text has a class', () => {
+    const icon = wrapper.find('i.fa-ban');
+    expect(icon).toHaveLength(1);
+  });
+
+  it('should have a button with a color class when the props color is set', () => {
+    wrapper.setProps({ color: 'primary' });
+    const span = wrapper.find('span.has-text-primary');
+    expect(span).toHaveLength(1);
+  });
+
+  it('should have a button with a size class when the props size is set', () => {
+    wrapper.setProps({ size: 'small' });
+    const span = wrapper.find('span.is-small');
+    expect(span).toHaveLength(1);
```

Comment:
In both previous cases, you only test when the value is set.

You don't test when it's not set 😄

Reply:
Missing the default test for this prop too

Reply:
> it('should have a button with no color and no size by default', () => {
>     const span = wrapper.find('span');
>    expect(span.prop('className')).toBe('icon');
> });
That tests the default props by default for both size and color
When I try to test separately size and color, I finished with the same test (only the name of the test was different)

Reply:
Aww yeah, didn't seen that 😛